script.js

```diff
@@ -1,10 +1,13 @@
 const startButton = document.querySelector('#startGameButton')
 const splashScreen = document.querySelector('#displaySplashInstructionsContainer')
 const avatar = document.getElementById('avatar')
+const gameTimerDisplay = document.querySelector('#stopwatchSecondsDisplay')
 
 avatar.style.left = '0px'
 let penguinPosition = parseInt(avatar.style.left)
 let lastKeyPressed = ""
+let timer = 0
+let raceStopwatch
 
 function checkKey(lastKeyPressed, currentKeyPressed) {
```

Comment:
the name of this function is a little confusing because it actually checks that the key is NOT the same as the last key pressed

maybe rename it to `checkKeyIsDifferent`